flink-clients/src/main/java/org/apache/flink/client/program/StandaloneClusterClient.java

```diff
@@ -70,6 +76,8 @@ public GetClusterStatusResponse getClusterStatus() {
 			} else {
 				throw new RuntimeException("Received the wrong reply " + result + " from cluster.");
 			}
+		} catch (FlinkRuntimeException | FlinkException e) {
```

Comment:
I think this catch branch is already covered by the catch branch for `Exception`.